staticd/static_routes.h

```diff
@@ -44,6 +44,26 @@ typedef enum {
 	STATIC_IPV6_GATEWAY_IFNAME,
 } static_types;
 
+/*
+ * Route Creation gives us:
+ *  START -> Initial State, only exit is when we send the route to
+ *          zebra for installation
+ * When we send the route to Zebra move to SENT_TO_ZEBRA
+ *  SENT_TO_ZEBRA -> A way to notice that we've sent the route to zebra
+ *                   But have not received a response on it's status yet
+ * After The response from zebra we move to INSTALLED or FAILED
+ *  INSTALLED -> Route was accepted
+ *  FAILED -> Route was rejected
+ * When we receive notification about a nexthop that a route uses
+ * We move the route back to START and initiate the process again.
+ */
+enum static_install_states {
+	STATIC_START,
+	STATIC_SENT_TO_ZEBRA,
+	STATIC_INSTALLED,
+	STATIC_NOT_INSTALLED,
+};
+
```

Comment:
I see where these states are being _set_, but I only see one place where one value is being used to change behavior. Is the 'state machine' something that's going to be a follow-on, maybe? If these are just tracking or informational values, maybe they could be in a show command somewhere?

Reply:
yeah.  The intent is to follow on with more debugging/show commands at some point in time in the future.  Right now static routes are a bit of a black box and you have to know what you are doing here to figure out.